app/ui/FormControlNumber.tsx

```diff
@@ -22,7 +22,7 @@ export function FormControlNumber(
         onChange(value);
       }}
       style={{
-        width: width !== undefined ? width : 160,
+        width: width ? width : 160,
```

Comment:
dunno what this change is for, but changing this causes an edge case that might be unintentional

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/30/commits/353662fe0ecc4d556ac0c63a4cdf14d6af011e9d